java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -1372,4 +1372,270 @@ CompletableFuture<GlideString[]> xclaimJustId(
+     *                     1L,
+     *                 }, {
+     *                     gs("1719710688676-0"),
+     *                     gs("Alice"),
+     *                     1719710718373L,
+     *                     1L
+     *                 } },
+     *             gs("consumers"), new Map[] {
+     *                 Map.of(
+     *                     gs("name"), gs("Alice"),
+     *                     gs("seen-time"), 1719710718373L,
+     *                     gs("active-time"), 1719710718373L,
+     *                     gs("pel-count"), 2L,
+     *                     gs("pending"), new Object[][] { {
+     *                             gs("1719710679916-0"),
+     *                             1719710707260L,
+     *                             1L,
+     *                         }, {
+     *                             gs("1719710688676-0"),
+     *                             1719710718373L,
+     *                             1L
+     *                         } }
+     *                 )
+     *             })
+     * }); // Detailed stream information for "my_stream".
+     * }</pre>
+     */
+    CompletableFuture<Map<GlideString, Object>> xinfoStreamFull(GlideString key);
+
+    /**
+     * Returns verbose information about the stream stored at key <code>key</code>.
+     *
+     * @since Redis 6.0 and above.
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param count The number of stream and PEL entries that are returned. Value of <code>0</code>
+     *     means that all entries will be returned.
+     * @return A <code>Map</code> of detailed stream information for the given <code>key</code>.
+     * @example
+     *     <pre>{@code
+     * // example of using the API:
+     * Map<String, Object> response = client.xinfoStreamFull("myStream", 42).get();
+     * }</pre>
+     *     The response has the same format as {@link #xinfoStreamFull(String)}.
+     */
+    CompletableFuture<Map<String, Object>> xinfoStreamFull(String key, int count);
+
+    /**
+     * Returns verbose information about the stream stored at key <code>key</code>.
+     *
+     * @since Redis 6.0 and above.
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param count The number of stream and PEL entries that are returned. Value of <code>0</code>
+     *     means that all entries will be returned.
+     * @return A <code>Map</code> of detailed stream information for the given <code>key</code>.
+     * @example
+     *     <pre>{@code
+     * // example of using the API:
+     * Map<GlideString, Object> response = client.xinfoStreamFull(gs("myStream"), 42).get();
+     * }</pre>
+     *     The response has the same format as {@link #xinfoStreamFull(GlideString)}.
+     */
+    CompletableFuture<Map<GlideString, Object>> xinfoStreamFull(GlideString key, int count);
```

Comment:
Can you please update all examples according to the comments I've left in this file? Thanks!